src/main/pages/setup-openshift/openshift-admin-guide.md

```diff
@@ -9,3 +9,76 @@ folder: setup-openshift
 
 {% include links.html %}
 {% include_relative admin-guide.md %}
+
+## Create workspace objects in personal namespaces
+
+When Che is installed on Openshift in multi-user mode, it is possible to register the Openshift server into the Keycloak server as an identity provider,
+in order to allow creating workspace objects in the personal OpenShift namespace of the user that is currenlty logged in Che through Keycloak.
+
+This feature is available only when Che is confirgured to create a new OpenShift namespace for every Che workspace.
+
+To enable this feature, the administrator should:
+- register, inside Keycloak, an Openshift identity provider that will point to the Openshift console of the cluster in which the workspace resources should be created,
+- configure Che to use this Keycloak identity provider in order to retrieve the Openshift tokens of Che users.
+
```

Comment:
Sorry, I know I approved this but in re-reading it I think we should be more clear on how to do these changes. Can we either write up the steps people need to do here explicitly or at least link off to specific pages of the Keycloak docs?  We need to assume the person using this doesn't know anything about keycloak, openshift or che.

Reply:
It turns out all the explanations are below, including screenshots

Reply:
Can we remove this section then and either (a) bring the more actionable section up or (b) provide a link down. We should avoid sections in the docs that talk generally about what you can do without being actionable.

Reply:
I see this as a summary of what needs to be done with detailed instructions below